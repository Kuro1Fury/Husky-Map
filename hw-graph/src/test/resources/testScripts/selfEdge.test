# selfEdge test
# Tests that a normal graph can be created, with a node n1. There is
# one edges, n1 to n1, to test whether self point edge is allowed.

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1

# Add one edges
AddEdge graph1 n1 n1 e1

# Print the nodes in the graph and the outgoing edges from n1 and n2
ListNodes graph1
ListChildren graph1 n1
