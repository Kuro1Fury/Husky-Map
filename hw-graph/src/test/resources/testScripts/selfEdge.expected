# selfEdge test
# Tests that a normal graph can be created, with a node n1. There is
# one edges, n1 to n1, to test whether self point edge is allowed.

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1

# Add one edges
added edge e1 from n1 to n1 in graph1

# Print the nodes in the graph and the outgoing edges from n1 and n2
graph1 contains: n1
the children of n1 in graph1 are: n1(e1)